### DISCUSSION TITLE
Save parking location

### DISCUSSION THREAD

**Discussion by ByteBitten** [organicmaps/organicmaps#1156] [Ideas]
Some GPS software have the option to save where you parked your car.
A simple button to add current location as "Parked here" to Bookmarks would make that easy.
Maybe with a little note option, like: "Level 3 spot 4".

**Comment by biodranik** [comment]
Apple and Google now do that automatically. I think it's the best way.

**Reply by ByteBitten** [reply]
That is nice and all, but that wouldn't work on a deGoogled phone...

**Reply by biodranik** [reply]
I mean it would be the best option to do the same in Organic Maps like Google and Apple are doing.

**Reply by ByteBitten** [reply]
I havn't yet experienced the automatic feature of how Google does it.
But adding a Save Parking to the My Location menu and/or Navigation Arival menu would work.
Storing it in Bookmarks under Parking category for finding it again.
If some implementation gets in, I'm already happy :)

**Reply by bentolor** [reply]
I'm also using exactly this usecase but my a manual approach: I have a bookmark collection "Parking" where i set a marker in foreign areas before going for a walk. 

Not sure if the proposed approach of an explicit button press brings that big UX benefit vs. complication. Rather if some automatic logic. But I doubt this is easily feasible w/o OS support like Googles way.
